packages/flutter/lib/src/material/time_picker.dart

```diff
@@ -202,12 +206,11 @@ class _TimePickerHeader extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    ThemeData theme = Theme.of(context);
-    TextTheme headerTheme = theme.primaryTextTheme;
-
+    ThemeData themeData = Theme.of(context);
```

Comment:
Yeah, we should pick one to go with.  `themeData` seems more accurate because it matches the type...